config/m4/fuse3.m4

```diff
@@ -30,42 +30,40 @@ AS_IF([test "x$with_fuse3" != "xno" -o "x$with_fuse3_static" != "xno"],
               ucx_fuse3_pkgconfig="$with_fuse3/lib$libsuff/pkgconfig"
               export PKG_CONFIG_PATH="$ucx_fuse3_pkgconfig:$PKG_CONFIG_PATH"])
 
-       FUSE3_CPPFLAGS=$(pkg-config --cflags-only-I fuse3)
-       FUSE3_LDFLAGS=$(pkg-config --libs-only-L fuse3)
```

Comment:
Does it work for fuse from user-defined location?
E.g. --with-fuse3=<path_to_fuse3>

Before your changes, when a user defines path to the fuse3, FUSE3_LDFLAGS contains path to libraries, and FUSE3_LIBS contains  libs. And now a user should add path to libraries to LD_LIBRARY_PATH. Right? Is that how it's meant to be?

Reply:
yes, there used same way as for direct call to pkg-config